apps/explorer-e2e/src/specs/address.spec.ts

```diff
@@ -14,90 +14,102 @@ import { exploredStabilization } from '../helpers/exploredStabilization'
 let explorerApp: ExplorerApp
 let cluster: Cluster
 
-test.beforeEach(async ({ page, context }) => {
-  cluster = await startCluster({ context })
-  await renterdWaitForContracts({
-    renterdNode: cluster.daemons.renterds[0].node,
-    hostdCount: cluster.daemons.hostds.length,
-  })
+const networkVersions = ['v1', 'v2', 'transition'] as const
```

Comment:
We can drop `transition`, it won't be any different from testing against v1 or v2 unless the test explicitly it written to mine through the network transition.